README.md

```diff
@@ -17,178 +17,45 @@ iDynTree is written in C++ language, but thanks to [SWIG](http://www.swig.org/)
 * **[Acknowledgments](#acknowledgments)**
 
 ## Installation
-iDynTree is mainly developed and mantained by the [iCub Tech facility](https://www.iit.it/research/facilities/icub-tech) and [Dynamic Interaction Control research line](https://www.iit.it/research/lines/dynamic-interaction-control) at the [Italian Institute of Technology](https://www.iit.it/), as part of the [iCub project](http://www.icub.org/) .
 
-For this reason it is usually installed through the [robotology-superbuild](https://github.com/robotology/robotology-superbuild), an easy way to download, compile and install the robotology software on multiple operating systems, using the [CMake](www.cmake.org) build system and its extension [YCM](http://robotology.github.io/ycm). To get iDynTree when using the `robotology-superbuild`, please enable the `ROBOTOLOGY_ENABLE_DYNAMICS` CMake option of the superbuild.
+### conda (recommended) 
 
-If you are not interested in installing all the robotology software it is still possible to install iDynTree without installing the rest of the robotology software, and please read the rest of the Readme for more info on this.
-
-### Dependencies
-iDynTree requires few external libraries. At the first configuration, the build system of iDynTree enables the use of the dependencies that it finds in the system,
-but you can manually make sure that iDynTree searches or ignores a given dependency by enabling or disabling the `IDYNTREE_USES_<DEP>` CMake options.
-
-##### Build dependencies
-- [CMake](http://www.cmake.org)
-
-##### Required
-- [Eigen](http://eigen.tuxfamily.org)
-- [Libxml2](http://xmlsoft.org/)
-
-##### Optional
-- [Assimp](http://www.assimp.org/)
-- [IPOPT](https://projects.coin-or.org/Ipopt)
-- [Qt5](https://www.qt.io/)
-- [YARP](https://github.com/robotology/yarp)
-- [ICUB](https://github.com/robotology/icub-main)
-- [irrlicht](http://irrlicht.sourceforge.net/)
-
-##### Optional for the optimal control part
-- [ALGLIB](https://github.com/S-Dafarra/alglib-cmake)
-- [osqp-eigen](https://github.com/robotology/osqp-eigen)
-- [WORHP](https://worhp.de/)
-
-#### Install dependencies
-
-If you need to install also `YARP` and `ICUB`, it is recommended that you install iDynTree via the [`robotology-superbuild`](https://github.com/robotology/robotology-superbuild). If instead you are not interested in the `YARP` and `ICUB` integration, you can easily install the rest of the dependencies using
-standard package managers.
-
-##### Windows
-
-On Windows we recommend to use  [`vcpkg`](https://github.com/Microsoft/vcpkg) C++ package manager to install iDynTree dependencies.
-
-###### vcpkg
-If you use [`vcpkg`](https://github.com/Microsoft/vcpkg), you can install all the required and optional dependencies of iDynTree using the following command:
+You can easily the library with [`conda`](https://github.com/conda-forge/osqp-eigen-feedstock) using the following command
```

Comment:
*install

Reply:
Thanks! Fixed in https://github.com/robotology/idyntree/pull/932/commits/ad39853abc47640540461e3b8707cc4a027f7278 .